x11/pango/Portfile

```diff
@@ -2,19 +2,19 @@
 
 PortSystem              1.0
 PortGroup               active_variants 1.1
-PortGroup               gobject_introspection 1.0
+PortGroup               meson 1.0
```

Comment:
As you moved to meson and removed the gobject_introspection you need to add that as a build dependency, or are you leaving that being added via harfbuzz

Reply:
Given the reasons for the gobject_introspection portgroup's existence, I think you might find that you need to add meson support to it rather than stop using it.

Reply:
I’d think it would be the other way around, adding gobject_introspection support into the meson portgroup since most protects that moved to meson need it.

There’s some Ports that use the meson portgroup and add in the needed functions to handle gobject_introspection usage.

Reply:
Sounds like further thought is needed, as both may be necessary. We'll figure it out longer-term, once we've finally gotten all of the umpteen Gimp dependencies - including these two - working and committed to master.

Reply:
I was going to submit a new ticket for this effort, but thankfully @kencu already took care of that:

[60644 - gobject_introspection portgroup: support meson](https://trac.macports.org/ticket/60644)